### PR TITLE
MC: add non-windows install to docs

### PR DESCRIPTION
**Pull request by black-sliver**

### CODE REVIEW

File: worlds/minecraft/docs/minecraft_en.md [hunk 1]
```diff
@@ -47,16 +47,29 @@ When the console tells you that you have joined the room, you're all set. Congra
 multiworld game! At this point any additional minecraft players may connect to your forge server. To start the game once
 everyone is ready use the command `/start`.
 
-## Manual Installation
+## Non-Windows Installation
+
+The Minecraft Client will install forge and the mod for other operating systems but Java has to be provided by the
+user. Head to [minecraft_versions.json](https://raw.githubusercontent.com/KonoTyran/Minecraft_AP_Randomizer/master/versions/minecraft_versions.json)
```
Comment (Ijwu) [inline worlds/minecraft/docs/minecraft_en.md:53]:
```suggestion
user. Head to [minecraft_versions.json on the MC AP GitHub](https://raw.githubusercontent.com/KonoTyran/Minecraft_AP_Randomizer/master/versions/minecraft_versions.json)
```

**Review by Ijwu — APPROVED** [review]
Optional change suggested to increase a11y of hyperlink

**Comment by black-sliver** [comment]
Still waiting for @ThePhar to validate instructions on macos

File: worlds/minecraft/docs/minecraft_en.md [hunk 2]
```diff
@@ -47,16 +47,29 @@ When the console tells you that you have joined the room, you're all set. Congra
...
 
 It is highly recommended to ues the Archipelago installer to handle the installation of the forge server for you.
-support will not be given for those wishing to manually install forge. For those of you who know how, and wish to do so,
+Support will not be given for those wishing to manually install forge. For those of you who know how, and wish to do so,
 the following links are the versions of the software we use.
 
-### Manual install Software links
+### Manual Installation Software Links
 
 - [Minecraft Forge Download Page](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.18.2.html)
```
Comment (ThePhar) [inline worlds/minecraft/docs/minecraft_en.md:70]: We should probably also remove the reference to a particular version of Forge as well, even if just in URL. Kono's `minecraft_versions.json` specifies the verified version of forge as well.

**Review by ThePhar — APPROVED** [review]
Verified it works on ARM MacOS so should also work on x86 MacOS. Do suggest the tweaks as well to the doc for Forge installation, but otherwise should be good to go.

**Pull request merged**